worlds/stardew_valley/strings/fish_names.py

```diff
@@ -147,6 +146,10 @@ class DistantLandsFish:
     giant_horsehoe_crab = "Giant Horsehoe Crab"
 
 
+class ModWaterItem:
+    dulse_seaweed = "Dulse Seaweed"

```

Comment:
Don't want to be super annoying, but If things should get split into mods, why not put the same SVE prefix as used in all other SVE groups?
```suggestion
class SVEWaterItem:
    dulse_seaweed = "Dulse Seaweed"
```